pkg/azuredisk/nodeserver_test.go

```diff
@@ -215,9 +215,10 @@ func TestNodeGetVolumeStats(t *testing.T) {
 	nonexistedPath := "/not/a/real/directory"
 	fakePath := "/tmp/fake-volume-path"
 	tests := []struct {
-		desc        string
-		req         csi.NodeGetVolumeStatsRequest
-		expectedErr error
+		desc         string
+		req          csi.NodeGetVolumeStatsRequest
+		expectedErr  error
+		skipOnDarwin bool
```

Comment:
@andyzhangx In many test cases, I have provided an option to skip on darwin as the tests return the following error:
`util/mount on this platform is not supported`